deployment/helm/values.yaml

```diff
@@ -91,7 +98,12 @@ agent:
     tag:
     # pullPolicy is the Akri Agent pull policy
     pullPolicy: ""
-  securityContext: {}
+  # ensures container doesn't run with unnecessary priviledges
+  securityContext:
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
```

Comment:
Tests are failing because tests are writing a file in the agent pod:
https://github.com/project-akri/akri/blob/47b42587620346e23258e19ed9c5c61ed6904487/test/run-end-to-end.py#L106

Reply:
Good point! Maybe we add that file as a volume mount for the Pod during the test

Reply:
I am not sure i understand how this was resolved. Looks like `readOnlyRootFilesystem: true` still exists which is ideal.

Reply:
I think the above highlight is outdated? [Here](https://github.com/project-akri/akri/blob/b4ffc1c39c52b84cadb3dcce0ed3ce2f9296bdb8/deployment/helm/values.yaml#:~:text=%3A%20%22%22-,securityContext%3A,drop%3A%20%5B%22ALL%22%5D,-host%3A) shows the updated security context for akri agent. I removed setting readOnlyRootFilesystem to true (followed what nvidia gpu device plugin did)